exercises/armstrong-numbers/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub fn is_armstrong_number(_num: u32) -> bool {
```

Comment:
Was there a particular reason to use `u32` instead of `usize`?

Reply:
It's strictly because of the tests. They overflow `u16` but could fit comfortably in `u32`